basler_camera:
  ros__parameters:
    publishers:
      raw_frame: /camera/raw_frame

    services:
      stop_camera: /camera/get_cam_state

    frames: 
      camera_link: camera_link
      
    rotation_angle: 0.0